okta/resource_okta_policy_password.go

```diff
@@ -219,6 +227,7 @@ func resourcePolicyPasswordRead(d *schema.ResourceData, m interface{}) error {
 		d.Set("password_max_lockout_attempts", policy.Settings.Password.Lockout.MaxAttempts)
 		d.Set("password_auto_unlock_minutes", policy.Settings.Password.Lockout.AutoUnlockMinutes)
 		d.Set("password_show_lockout_failures", policy.Settings.Password.Lockout.ShowLockoutFailures)
+		d.Set("lockout_notifications", policy.Settings.Password.Lockout.UserLockoutNotificationChannels)
```

Comment:
Since this is a set you will need to convert it, try this 
```
d.Set("lockout_notifications", convertStringSetToInterface(policy.Settings.Password.Lockout.UserLockoutNotificationChannels))
```